django_x509/base/admin.py

```diff
@@ -175,6 +186,16 @@ def revoke_action(self, request, queryset):
 
     revoke_action.short_description = _('Revoke selected certificates')
 
+    def renew_cert(self, request, queryset):
+        renewed_rows = 0
+        for cert in queryset:
+            cert.ca.renew_ca()
```

Comment:
why are you renewing the CA? In this case we have to renew only the certificate.